Source/IntegrationTests/TestFiles/LitTests/LitTest/dafny0/snapshots/Snapshots2.run.legacy.dfy.expect

```diff
@@ -19,11 +19,11 @@ Processing implementation Q (well-formedness) (at Snapshots2.v1.dfy(13,11)):
 Processing command (at Snapshots2.v1.dfy(4,10)) assert {:id "id12"} Lit(false);
   >>> DoNothingToAssert
 Snapshots2.v1.dfy(4,9): Error: assertion might not hold
-Processing command (at Snapshots2.v1.dfy(11,11)) assert {:id "id16"} true;
+Processing command (at Snapshots2.v1.dfy(11,11)) assert {:id "id16"} Lit(true);
```

Comment:
We are lucky that it's the only difference. Since a Dafny literal is wrapped with Lit, I thought that creating a Dafny expression instead of a Boogie expression would create more differences.
Hopefully it's negligible for verification.

Reply:
Yeah, I was expecting that might happen, too. I also think this change will be reverted when we merge #4596.